client/state/ui/editor/actions.js

```diff
@@ -82,7 +84,7 @@ export function startEditingNewPost( siteId, post ) {
  */
 export function stopEditingPost( siteId, postId ) {
 	return dispatch => {
-		dispatch( editorReset() );
+		dispatch( editorReset( { isLoaded: false } ) );
```

Comment:
This change can be reverted now, can't it? The `isLoaded` flag doesn't do anything and the "iframe loaded" state is set back to `false` somewhere else?

Reply:
Yep - that's correct. I meant to remove these references so not sure how I missed that. Thanks!

Reply:
Fixed in a0c2e4e